src/main/java/io/codelex/Test/Exercise1/CreditCard.java

```diff
@@ -0,0 +1,27 @@
+package io.codelex.Test.Exercise1;
+
+import java.math.BigDecimal;
+
+public class CreditCard extends Card{
+
+    public CreditCard(String number, String owner, int CCVcode, int balance) {
+        super(number, owner, CCVcode, balance);
+
+    }
+
+    @Override
+    public void addMoney(int add) {
```

Comment:
Ja metode dara 1:1 ko parent klases metode (vai arī tikai izsauc parent klases metodi), tad no viņas nav nozīmes.
Ja šī metode nebūtu dzēsta, tad new CreditCard(...).addMoney(1) izdarītu tieši to pašu, ko ja metode būtu dzēsta.